services/core-web/src/components/noticeOfWork/applications/referals/ReferralConsultationPackage.js

```diff
@@ -225,9 +226,16 @@ export class ReferralConsultationPackage extends Component {
 
   render() {
     const label = this.props.type === "REF" ? "Referral Package" : "Consultation Package";
-    const disabled = !this.props.progress[this.props.type];
+    const disabled =
+      !isEmpty(this.props.progress[this.props.type]) &&
+      !isNil(this.props.progress[this.props.type].end_date);
+
```

Comment:
Can you move this check back into a const?
```
const complete = !isEmpty(this.props.progress[this.props.type]) &&
      !isNil(this.props.progress[this.props.type].end_date);

const disabled = complete;
```
It will make it easier in the future to understand what's going on here.

Reply:
Sounds good, it is added for future reference.